portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -16,20 +16,45 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Arrays;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   static final long serialVersionUID = 1L;
 
+  /**
+   * Constant test comments
+   */
+  static final String[] TEST_COMMENTS = {
+    "This is a normal comment",
+    "This is another normal comment",
+    "This is yet a third comment"
+  };
+  
+  private ArrayList<String> commentsList =
+    new ArrayList<String>(Arrays.asList(TEST_COMMENTS));
```

Comment:
Should not need to wrap the `Arrays.asList` in second constructor. I'm assuming you were getting an error otherwise?

You can force the type via `Arrays.<String>(TEST_COMMENTS)`

Reply:
Arrays.<String>asList() returns a List<String>, whereas commentsList is of type ArrayList<String>. Since ArrayList is a subclass of List, I cannot assign the result directly to commentsList and thus I have to "convert" the List<String> into an ArrayList<String> through the second constructor.